lib/Driver/Driver.cpp

```diff
@@ -194,6 +194,14 @@ class Driver::InputInfoMap
 };
 using InputInfoMap = Driver::InputInfoMap;
 
+static bool failedToReadOutOfDateMap(bool ShowIncrementalBuildDecisions) {
+  if (ShowIncrementalBuildDecisions) {
+    llvm::outs() << "Incremental compilation has been disabled due to a "
+                    "malformed build record file.\n";
+  }
```

Comment:
To ease debugging, I think it'd be great to print the path to the build record file by passing `buildRecordPath` to this function. What do you think?